### ISSUE TITLE
Improvements in labeling tip

### ISSUE THREAD

**Issue by infosquatter** [globaleaks/GlobaLeaks#1453]
I know it is an improvement of another issue, but I could not find it - sorry.

We tested the new custom label for tips feature during Mexicoleaks bootcamp and it lacks some very important things.

* These labels should viewed by all the receptors. So if receptor A and B received the same material and A added a label, it should be viewed also by B.

* The labels should be viewed in the receptor page which summarizes the tips. So it could get a glance on the tip without entering into the tip interface.

* Multiple labels should be possible.

I believe the important part of this tip - before helping a receiver to categorize its tips - is to facilitate collaboration among receptors while working on the same tip.

**Comment by evilaliv3** [comment]
@infosquatter thins are exactly by design as you are reporting:
- each recipient can label its own tip.
- each recipien can see only own label
- receivers see each own label on submission list

i wonder why this ticket. probably you using only one receiver or what?

**Comment by infosquatter** [comment]
@evilaliv3 no, I think you did not understand or I expressed myself wrongly: the points I mentioned **are missing**. Do you understand?

*  each recipien can see only own label --- -- we need recipients to see each other labels.

* multiple labels.

* these labels to be all shown in the submission list.

**Comment by fpietrosanti** [comment]
Adding in the title to "+Multiple Labels +Visible to Multiple Receivers"

**Comment by infosquatter** [comment]
I would strongly vote for the second 

> all receivers are indepdendent (nothing shared)

**Comment by evilaliv3** [comment]
@infosquatter: i think for various reasons we would require both the models; i was simply clarifying that in my opinion what we should avoid is a mixed approach to keep a much simple as possible.
